.github/workflows/_github-release.yml

```diff
@@ -56,7 +59,7 @@ jobs:
           echo The last commit sha in the release: ${{ inputs.sha }}
           echo The release version number: ${{ inputs.version_number }}
           echo Expected Changlog path: ${{ inputs.changelog_path }}
-          echo testing: ${{ inputs.testing }}
+          echo Test run: ${{ inputs.test_run }}
```

Comment:
As part of the cleanup, please update these kinds of blocks so the outputs all line up.

For example:
```
echo The last commit sha in the release:  ${{ inputs.sha }}
echo The release version number:          ${{ inputs.version_number }}
echo Expected Changlog path:              ${{ inputs.changelog_path }}
echo Test run:                            ${{ inputs.test_run }}
```

Reply:
Fixed it.